.cci.jenkinsfile

```diff
@@ -25,17 +25,7 @@ pod(image: imageName + ":latest", kvm: true, memory: "10Gi") {
     }
 
     stage("Test Live Images") {
-        try {
-            parallel metal: {
-                shwrap("cd /srv && env TMPDIR=\$(pwd)/tmp/ kola testiso -S --output-dir tmp/kola-testiso-metal")
-            }, metal4k: {
-                shwrap("cd /srv && env TMPDIR=\$(pwd)/tmp/ kola testiso -S --output-dir tmp/kola-testiso-metal4k --no-pxe --qemu-native-4k")
-            }
-        } finally {
-            shwrap("cd /srv && tar -cf - tmp/kola-testiso-metal/ | xz -c9 > ${env.WORKSPACE}/kola-testiso-metal.tar.xz")
-            shwrap("cd /srv && tar -cf - tmp/kola-testiso-metal4k/ | xz -c9 > ${env.WORKSPACE}/kola-testiso-metal4k.tar.xz")
-            archiveArtifacts allowEmptyArchive: true, artifacts: 'kola-testiso*.tar.xz'
-        }
+        fcosKolaTestIso(cosaDir: "/srv", extraArgs4k: "--no-pxe")
```

Comment:
`fcosKolaTestIso` already wraps the run with a `stage()`, so we can drop the outer `stage` here.

Reply:
@jlebon Thanks =). Fixed